frontend/src/components/MultiSelection.tsx

```diff
@@ -7,53 +7,38 @@ import {
   SelectOptionObject,
   SelectVariant,
 } from '@patternfly/react-core';
-import { MenuItemStatus } from 'pages/groupSettings/GroupTypes';
+import { MenuItemStatus } from 'pages/groupSettings/groupTypes';
 
-type MenuOptionMultiSelectProps = {
-  initialState: MenuItemStatus[];
-  onChange: (itemSelection: MenuItemStatus[]) => void;
+type MultiSelectionProps = {
+  value: MenuItemStatus[];
+  setValue: (itemSelection: MenuItemStatus[]) => void;
+  clearSelection: () => void;
 };
 
-const ARIAL_LABEL = 'Select a group';
-const ERROR_LABEL = 'One or more group must be selected';
-
-export const MenuOptionMultiSelect: React.FC<MenuOptionMultiSelectProps> = ({
-  initialState,
-  onChange,
+export const MultiSelection: React.FC<MultiSelectionProps> = ({
+  value,
+  setValue,
+  clearSelection,
 }) => {
-  const [menuItemsState, setMenuItemsState] = useState<MenuItemStatus[] | undefined>();
-  const [showMenu, setShowMenu] = useState<boolean>(false);
-
-  useEffect(() => {
-    setMenuItemsState(initialState);
-  }, [initialState]);
+  const [showMenu, setShowMenu] = React.useState<boolean>(false);
 
   const toggleMenu = (isOpen: React.SetStateAction<boolean>) => {
     setShowMenu(isOpen);
   };
 
   const onSelect = (
     _: React.MouseEvent<Element, MouseEvent> | React.ChangeEvent<Element>,
```

Comment:
fwiw, if you inline this function, TS will infer the types of the properties from the component Type and you don't have to type the event object.